msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -1524,6 +1524,131 @@ The following code provides an example of meeting end event payload:
 | **value.EndTime** | The meeting end time in UTC. |
 | **locale**| The locale of the message set by the client. |
 
+## Receive meeting participant events
+
+Your bot can receive real-time meeting details such as participant join and leave events through the meeting event subscription in Developer Portal. A bot receives the participant events only if subscribed to these events in Developer Portal.
```

Comment:
Should be: Your bot can receive real-time meeting **events** such as ....

Reply:
Updated